terraform/031-email-service/vars.tf

```diff
@@ -160,3 +160,15 @@ variable "wildcard_cert_arn" {
 variable "enable_cron" {
   default = true
 }
+
+variable "app_id" {
```

Comment:
It is partially because they are used in so many places, and because they are interpolated into the policy on line 130 (and equivalent in other modules) a longer name seemed unwieldy. I understand the tradeoff, though, so I did wrestle over the naming of these variables. In the end, I went with the shorter name because of the fact that they are somewhat ubiquitous and confusion should be less because of how prominent they are.